src/matlab/setopengl.m

```diff
@@ -22,9 +22,9 @@ function setopengl()
 for i = 1:length(rset),
   if (strcmp(lower(rset(i)),'opengl'))
     found_opengl = 1;
-    break;
-  end;
-end;
+    break
```

Comment:
Yes, but it is nested in a for-loop, so it is okay, and the right thing to do, since I don't wan to return from the function, only break out of the for-loop.